managed-services/webhosting/quickstart.mdx

```diff
@@ -0,0 +1,65 @@
+---
+meta:
+  title: Webhosting - Quickstart
+  description: This page shows you how to get started with Webhosting.
+content:
+  h1: Webhosting - Quickstart
+  paragraph: This page shows you how to get started with Webhosting.
+tags: webhosting cpanel
+dates:
+  validation: 2022-11-24
+  posted: 2021-05-26
+categories:
+  - webhosting
+---
+cPanel is a website management control panel tool, providing users with a graphical interface and fast access icons which allow the configuration and monitoring of their hosting solutions. Features like publishing websites, launching databases, creating emails, and managing files are available through the cPanel interface with elevated performance.
+
+## Getting started
+
+Not sure where to begin? Take a look at our Frequently Asked Questions, which guide you to the relevant cPanel documentation pages.
+
+* [How do I use cPanel?](https://docs.cpanel.net/cpanel/the-cpanel-interface/the-cpanel-interface/)
+* [How do I set up SSH Access?](https://docs.cpanel.net/cpanel/security/ssh-access/)
+* [How do I manage my e-mail accounts?](https://docs.cpanel.net/cpanel/email/email-accounts/#overview)
+* [How do I configure my FTP accounts (create, access & delete)?](https://docs.cpanel.net/cpanel/files/ftp-accounts/)
+* [How do I manage my databases?](https://docs.cpanel.net/cpanel/databases/)
+  * [With MySQL](https://docs.cpanel.net/cpanel/databases/mysql-databases/)
+  * [With PostgresQL](https://docs.cpanel.net/cpanel/databases/postgresql-databases/)
+* [How do I manage my domains?](https://docs.cpanel.net/cpanel/domains/domains/)
+
+## How to access cPanel via the Console 
+
+1. Click **Webhosting** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu. The **Webhosting** overview page displays.
+2. Click <icon name="seeMore" /> or the domain name of the webhosting you want to configure. The **Hosting information** page displays.
+3. Click **Dashboard** next to **Link to cPanel** to open the cPanel interface in a new tab.
+    <Lightbox src="scaleway-webhosting-cpanel.webp" alt="" />
+    <Message type="tip">
+        Alternatively you can reach cPanel via the **cPanel** button in the top right corner of the **Hosting information** section.
+    </Message>
+4. Enter your cPanel credentials and click **Log in** to log into cPanel. 
+
+## How to access webmail 
+
+If you wish to manage your e-mails through cPanel, you can access the webmail platform directly via the webmail link in the **Hosting information** of your Webhosting.
+
+## How to configure DNS for Webhosting
+
+To link your domain name with your Webhosting you have to update the nameservers in the DNS configuration at your registrar. Nameservers are servers used to translate domain names into IP addresses. When you enter a URL in a browser, they show you where the website is located, instead of memorizing entire numerical sequences in the form of IP addresses.
```

Comment:
```suggestion
To link your domain name with your Webhosting service you have to update the nameservers in the DNS configuration at your registrar. Nameservers are servers used to translate domain names into IP addresses. When you enter a URL in a browser, they show you where the website is located, instead of memorizing entire numerical sequences in the form of IP addresses.
```